client/src/pages/ModalBookingScreen.js

```diff
@@ -12,11 +12,23 @@ import {
 	Button,
 	Header,
 	Modal,
+	Dropdown,
 } from "semantic-ui-react";
 
+const deskSelection = [
+	{ key: 1, text: "Desk 1", value: 1 },
+	{ key: 2, text: "Desk 2", value: 2 },
+	{ key: 3, text: "Desk 3", value: 3 },
+];
+
 function ModalBookingScreen({ bookingDate }) {
 	const [open, setOpen] = React.useState(false);
 	const [name, setName] = useState("");
+	const [checked, setChecked] = useState(false);
+
+	const handleCheck = () => {
+		setChecked(!checked);
```

Comment:
when setting a useState, if you need to use the current value, you need a handler function:

setChecked((previousChecked) => !previousChecked)

The exact reasons why are complicated, but it's just a good thing to know

Reply:
alternatively, you can use the data from the event of the checkbox itself, see below